main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
- 
+//  Noun:       bomb
+//  action 1:   the bomb explodes
+bomb.detonate();
+//  action 2:   the bomb arms
+bomb.arm();
+//  action 3:   the bomb disarms
+bomb.disarm();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       phone
+//  action 1:   the phone dials the number on display
+phone.callNumber();
+//  action 2:   the phone terminates the call
+phone.hangup();
+//  action 3:   the phone sends the text message
+phone.sendTxt(); 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       dog
+//  action 1:   the dog takes a drink
+dog.drink();
+//  action 2:   the dog eats any nearby food
+dog.eat();
+//  action 3:   the dog makes the feces
+dog.makePoop(); 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       car       
+//  action 1:   the car starts the engine
+car.startEngine();
+//  action 2:   the car opens the trunk
+car.openTrunk();
+//  action 3:   the car applies the brakes
+car.applyBrakes(); 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       tv
+//  action 1:   the TV changes current channel
+tv.changeChannel();
+//  action 2:   the TV mutes the sound
+tv.mute()
+//  action 3:   the TV turns off
+tv.turnOff(); 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       bird
+//  action 1:   the bird takes off in flight
+bird.fly();
+//  action 2:   the bird builds a nest
+bird.makeNest();
+//  action 3:   the bird eats any nearby worm
+bird.eatWorm(); 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       guitarplayer
+//  action 1:   guitarplayer plays face-melting solo
+guitarplayer.playSolo();
+//  action 2:   guitarplayer distorts face because badass
+guitarplayer.makeWierdFace();
+//  action 3:   guitarplayer antagonizes keyboard player
+guitarplayer.talkSmackToKeyboardPlayer(); 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       keyboardplayer
+//  action 1:   keyboardplayer unplugs guitarplayer from amp
+keyboardplayer.unplugGuitarPlayer();
+//  action 2:   keyboardplayer evades fists from guitarplayer
+keyboardplayer.takeEvasiveAction();
+//  action 3:   keyboardplayer protects expensive gear from melee
+keyboardplayer.protectGear();
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       drummer
+//  action 1:   drummer strikes drum head
+drummer.bangDrum()
+//  action 2:   drummer twirls sticks around fingers
+drummer.twirlSticks();
+//  action 3:   drummer throws sticks out to crowd
+drummer.throwSticksIntoAudience();
 //  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       fan
+//  action 1:   fan screams with delight
+fan.scream();
+//  action 2:   fan makes picking motion and waits for flying picks
+fan.retrievePicksThrownByGuitarPlayer();
+//  action 3:   fan pulls out lighter (fan is old fart)
+fan.deployLighter();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.